### ISSUE TITLE
RightCurly: multiline try-with-resources gives false positive

### ISSUE THREAD

**Issue by mhendriks91** [checkstyle/checkstyle#3789]
/var/tmp $ cat Main.java

```
public class Main {
    public void method() {
        try (AutoCloseable x = null;
             AutoCloseable y = null) {
            // Body
        } 
        catch (Exception e) {
            // Exception handling
        }
    }
}
```

/var/tmp $ cat config.xml

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN" "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="RightCurly"/>
    </module>
</module>
```

/var/tmp $ java -jar checkstyle-7.5-all.jar -c config.xml Main.java

```
[ERROR] /var/tmp/Main.java:4:36: '}' at column 36 should be on a new line. [RightCurly]
```

---------------

The code should be accepted. There is not even a right curly on line 4 (column 36 contains a closing parenthesis), and every right curly in the file is on its own line. This occurs only on a multiline try-with-resources statement (multiple variable declarations in the try's body). Whether or not there is a catch present does not matter.

**Comment by rnveach** [comment]
duplicate of #3776 , the issue was fixed and will be in hot-fix release.
